src/emmy/expression/compile.cljc

```diff
@@ -296,61 +429,78 @@
   (sci/eval-form (sci/fork sci-context) f-form))
 
 (defn- compile->clj
-  "Returns Clojure source for a function that implements `body`, given:
+  "Returns Clojure source for a function that implements `body`, given
+  a map with keys:
 
-  - `argv`: a vector of symbols to serve as the function's arguments.
+  - `:argv`: a vector of symbols to serve as the function's arguments.
     The vector may be nested for sequence destructuring.
 
-  - `body`: a function body making use of any symbol in argv above"
-  [argv {:keys [bindings body]}]
-  (w/postwalk-replace
-   sym->resolved-form
-   (if (seq bindings)
-     `(fn [~@argv]
-        (let ~(into [] cat bindings)
-          ~body))
-     `(fn [~@argv]
-        ~body))))
+  - `:body`: a function body making use of any symbol in argv above"
+  [{:keys [argv body]}]
+  (w/postwalk-replace sym->resolved-form `(fn [~@argv] ~body)))
```

Comment:
`sym->resolved-form` smells like the next keyword option, probably for another PR. Some other dictionaries of math that I would use:

- https://github.com/generateme/fastmath/blob/master/src/fastmath/core.clj
- the `emmy.generic` functions, if, for example, I wanted to send a function across the wire and then compile it back to generic
- this is how we might think about compiling, say, matrix-valued functions (that wouldn't work with the primitive calling convention, but it would be great with the other options!)